includes/admin/class-wc-stripe-inbox-notes.php

```diff
@@ -12,9 +12,17 @@
  * @since 4.5.4
  */
 class WC_Stripe_Inbox_Notes {
-	const NOTE_NAME = 'stripe-apple-pay-domain-verification';
+	const SUCCESS_NOTE_NAME = 'stripe-apple-pay-marketing-guide-holiday-2020';
+	const FAILURE_NOTE_NAME = 'stripe-apple-pay-domain-verification-needed';
 
-	public static function notify_of_apple_pay_domain_verification_if_needed() {
+	public function __construct() {
+		add_action( 'wc_stripe_apple_pay_post_setup_success', array( self::class, 'create_marketing_note' ) );
```

Comment:
We use wc_stripe_apple_pay_post_setup_success in three places. Please make it a constant instead for safety's sake.

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1365/commits/a12265c64c4e661b63b9ca11e90d3279c97c769c